### ISSUE TITLE
Visible labels are not being imported from racktables

### ISSUE THREAD

**Issue by imriz** [device42/Racktables-to-Device42-Migration#3]
We didn't notice any built in field for this in Device42, so we suggest using custom field, and populate that during migration.

**Comment by rickjohnston** [comment]
Hi @imriz can you please share an example?

**Comment by imriz** [comment]
@rickjohnston Hi :)
Example of what?

**Comment by rickjohnston** [comment]
Let me rephrase - we are not fully familiar with racktables, so which object/DB field has this label and which object/field in D42 should have it?

**Comment by imriz** [comment]
Every type of object has it, this is a column in the Object table.

**Comment by imriz** [comment]
@rickjohnston we will open a pull request with our suggested solution

**Comment by rickjohnston** [comment]
Sounds good @imriz Also, is it something like tags in D42 or different?

**Comment by imriz** [comment]
@rickjohnston different, I think (we're new to device42 - considering it as a replacement for racktables, obviously :) )

In our use case, this describes the physical visible label attached to device, used by on premises personnel to identify the device. The description field ("name") is used to describe the logical name of the device.

**Comment by rickjohnston** [comment]
I see. Thinking out loud, it might be best handled by using some sort of mapping in the conf file when setting up the migration. For example, for a device a physical visible label could be asset tag (asset_no in D42) or it could be a custom field, if asset tag and label are handled differently. It will depend on how a particular environment is setup. Also, if using custom fields, what custom field name to use for that physical label should go in the mapping file.

**Comment by imriz** [comment]
We will go with whatever you suggest - what do you think is the best field for a visible label?

**Comment by rickjohnston** [comment]
It might be better solved by understanding your current setup. I recommend shooting an email to support@device42.com and setting up a quick phone call or a web meeting. That said, here are my thoughts(which is from a limited understanding based on the conversation above and your other pull request for asset_no to serial_no mapping https://github.com/device42/Racktables-to-Device42-Migration/issues/4  ): 

In your scenario:
asset_no = serial_no in D42
physical label = asset_no in D42 (not 100% sure about this).

If that is how you are setup, I think best approach would be for us to modify the mapping file in the migration script so you can make that mapping.

**Comment by osanchez42** [comment]
If there are any requests or support is needed please contact Device42 using the contact methods listed at https://www.device42.com/contact/
